### PR TITLE
qa: ignore container checkpoint/restore related selinux denials for container-hosts/centos9

### PR DESCRIPTION
**Pull request by vshankar**
Fixes: http://tracker.ceph.com/issues/66640

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [ ] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

**Review by adk3798 — APPROVED** [review]

**Comment by adk3798** [comment]
jenkins test make check

**Review by batrick — APPROVED** [review]

File: qa/distros/container-hosts/centos_9.stream.yaml [hunk 1]
```diff
@@ -4,4 +4,4 @@ overrides:
   selinux:
     allowlist:
       - scontext=system_u:system_r:logrotate_t:s0
-
+      - scontext=system_u:system_r:getty_t:s0
```
Comment (batrick) [inline qa/distros/container-hosts/centos_9.stream.yaml:7]: nit: newline would be nice here

**Comment by batrick** [comment]
(adding this to my future run but feel free to test this in others.)

**Comment by batrick** [comment]
jenkins test make check arm64

**Comment by batrick** [comment]
needs rebase

File: qa/distros/container-hosts/centos_9.stream.yaml (hunk 1 above)
Comment (vshankar) [reply qa/distros/container-hosts/centos_9.stream.yaml:7]: ack

**Comment by vshankar** [comment]
rebased and updated

**Comment by vshankar** [comment]
This PR is under test in [https://tracker.ceph.com/issues/66850](https://tracker.ceph.com/issues/66850).

**Review by rishabh-d-dave — APPROVED** [review]
Perhaps we should bit of context to commit message about why we need to add it allowlist.

**Comment by vshankar** [comment]
https://tracker.ceph.com/projects/cephfs/wiki/Main#wip-vshankar-testing-20240705161036-debug

**Comment by vshankar** [comment]
jenkins retest this please

**Comment by vshankar** [comment]
jenkins test api

**Pull request merged**